torchgeo/models/weights.py

```diff
@@ -0,0 +1,154 @@
+"""Pretrained weights for models."""
+import os
+from typing import Any, Dict
+
+import torch
+import torch.nn as nn
+from torchvision.models._api import Weights, WeightsEnum
+
+from torchgeo.datasets.utils import download_url
+
+__all__ = ["ResNet50_Weights"]
+
+
+class ResNet50_Weights(WeightsEnum):
+    """ResNet50 weights.
+
+    For `timm
+    <https://github.com/rwightman/pytorch-image-models>`_ implementation.
+    """
+
+    IMAGENET_RGB = Weights(
+        url=(
+            "https://github.com/rwightman/pytorch-image-models/releases/"
+            "download/v0.1-rsb-weights/resnet50_a1_0-14fe96d1.pth"
+        ),
+        transforms=nn.Identity(),
+        meta={
+            "dataset": "imagenet",
+            "ssl_method": None,
+            "repo": (
+                "https://github.com/rwightman/pytorch-image-models/"
+                "blob/main/timm/models/resnet.py"
+            ),
+        },
+    )
+    SENTINEL2_RGB_SECO = Weights(
+        url="https://zenodo.org/record/4728033/files/seco_resnet50_1m.ckpt?download=1",
+        transforms=nn.Identity(),
+        meta={
+            "ssl_method": "seco",
+            "dataset": "seco",
+            "publication": "https://arxiv.org/abs/2103.16607",
+            "repo": "https://github.com/ServiceNow/seasonal-contrast",
+        },
+    )
+
+    AERIAL_MILLIONAID_RGB = Weights(
```

Comment:
What does AERIAL mean here? It would be good to have some unique identifier that describes the exact sensor since different planes may have different # channels or resolution

Reply:
The weights are pretrained on  [MillionAid dataset](https://captain-whu.github.io/DiRS/) and come from GoogleEarth so would `GoogleEarth` be better?